app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```

Comment:
Like we talked about yesterday, I think this is great for our MVP. I'd like to see an update to this if we have time though that allows you to put your username in and then the email gets sent to the email address associated with that username for the added level of security

Reply:
I'll make an issue for our project board so we can remember to circle back to it.